Upstream: no, Exherbo specific

From a33007119bd613538da21bcb7f98da881be0b22e Mon Sep 17 00:00:00 2001
From: Ingmar Vanhassel <ingmar@exherbo.org>
Date: Tue, 11 Nov 2014 18:29:38 +0100
Subject: [PATCH] Don't pass CFLAGS etc to install target

They make my build fail at least:

=== Starting src_install
<SNIP>
/bin/sh ./Build  -march=native -O2 -pipe install
Absolute directory path required for -O flag
Makefile:14: recipe for target 'install' failed

---
 libmilter/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmilter/Makefile b/libmilter/Makefile
index 825ddd5..8fa445a 100644
--- a/libmilter/Makefile
+++ b/libmilter/Makefile
@@ -11,7 +11,7 @@ check: FRC
 clean: FRC
 	$(SHELL) $(BUILD) $(OPTIONS) $@
 install: FRC
-	$(SHELL) $(BUILD) $(OPTIONS) $@
+	$(SHELL) $(BUILD) $@
 
 fresh: FRC
 	$(SHELL) $(BUILD) $(OPTIONS) -c
-- 
2.1.3

